bundle/manifests/metallb-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,382 @@
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        - apiGroups:
+          - policy
+          resourceNames:
+          - controller
+          resources:
+          - podsecuritypolicies
+          verbs:
+          - use
+        serviceAccountName: controller
+      - rules:
+        - apiGroups:
+          - metallb.io
+          resources:
+          - addresspools
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - metallb.io
+          resources:
+          - addresspools/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - metallb.io
+          resources:
+          - metallbs
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - metallb.io
+          resources:
+          - metallbs/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - policy
+          resources:
+          - podsecuritypolicies
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: default
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - endpoints
+          - nodes
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - discovery.k8s.io
+          resources:
+          - endpointslices
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        - apiGroups:
+          - policy
+          resourceNames:
+          - speaker
+          resources:
+          - podsecuritypolicies
+          verbs:
+          - use
+        serviceAccountName: speaker
+      deployments:
+      - name: metallboperator-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                control-plane: controller-manager
+            spec:
+              containers:
+              - args:
```

Comment:
do we need the rbac-proxy? I think we are not exposing anything from the operator right now maybe we can add this in the future

Reply:
i think it was added automatically in the initial setup, should we remove it?
https://github.com/metallb/metallb-operator/blob/main/config/default/manager_auth_proxy_patch.yaml

Reply:
if you are not using rbac proxy yes